simplesat/rules_generator.py

```diff
@@ -1,9 +1,8 @@
 import collections
 import enum
 
-from enstaller.errors import EnstallerException
-
 from .constraints import Requirement
+from .errors import NoPackageFound, SolverException
```

Comment:
Does this trip up higher level tools that would be using this in place of enstaller? They will be expecting the other class object with this name.

Not sat solver's fault, but now we'll have to import and catch both sets of exception classes, right?